### PR TITLE
setup project in eclipse and added appropriate entries in gitIgnore

### PR DESCRIPTION
**Pull request by bart-de-jonge**
also changed mainClass to not give warnings

### CODE REVIEW

**Comment by Markvanderuit** [comment]
Uhh, seems good, I guess?

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,9 @@
 out/
 site/
 ContextProject.iml
+/logs/*
+.classpath
+.project
+.settings/org.eclipse.jdt.core.prefs
```
Comment (martijnjanssen) [inline .gitignore]: Why not ignore the entire folder?

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,9 @@
 out/
 site/
 ContextProject.iml
+/logs/*
```
Comment (martijnjanssen) [inline .gitignore]: Is there a specific reason for the slash in front?

Comment (bart-de-jonge) [inline .gitignore]: no there is no specific reason

File: .gitignore (hunk 1 above)
Comment (bart-de-jonge) [inline .gitignore]: This was used in another project from which I copied it, if you can state things more concise, please edit it in a future commit

**Comment by bart-de-jonge** [comment]
Nice work, To me everything seems fine

**Comment by mennoo1996** [comment]
Well done!

**Pull request merged**